pkg/restapi/issuer/operation/operations.go

```diff
@@ -576,6 +606,34 @@ func (o *Operation) createReferenceCredential(token string, profile *issuer.Prof
 	return vcBytes, nil
 }
 
+func (o *Operation) createOIDCClient(profileData *issuer.ProfileData) (oidcClient, error) {
+	if client, present := o.cachedOIDCClients[profileData.ID]; present {
+		return client, nil
+	}
+
+	secret := make([]byte, 256)
```

Comment:
seems like this secret is:

a) not persisted anywhere, and
b) not shared with anyone

Reply:
I'm still trying to work out how the authorization server should provide the client secret - it needs to be provided as part of registering the issuer-adapter to the issuer.